library/std/src/sys/pal/windows/thread.rs

```diff
@@ -115,6 +115,14 @@ impl Thread {
     }
 }
 
+pub(crate) fn current_os_id() -> Option<u64> {
+    // SAFETY: FFI call with no preconditions.
+    let id: u32 = unsafe { c::GetThreadId(c::GetCurrentThread()) };
```

Comment:
I am going to change this to `GetCurrentThreadId` that does both steps, but not until the Miri shims merge so I can stop manually syncing changes.